src/js/libgeo/StateIO.js

```diff
@@ -105,6 +110,10 @@ function saveGeoElement(el) {
         if (val !== null && val !== undefined)
             res[key] = val;
     });
+    if (el.kind === "P" && (!el.movable || el.pinned) && res.color) {
```

Comment:
This is the counterpart for [the dimming in `pointDefault`](https://github.com/gagern/CindyJS/blob/85374cf4386361c7f7004ba6fa48d532ce81f93e/src/js/libgeo/GeoBasics.js#L77-L79).